libs/features/personalization/personalization.js

```diff
@@ -606,6 +606,7 @@ function compareExecutionOrder(a, b) {
 export function cleanAndSortManifestList(manifests) {
   const manifestObj = {};
   manifests.forEach((manifest) => {
+    if (!manifest) return;
```

Comment:
What's the use-case here? Can `manifests` contain falsy values?

Reply:
yes, my sample page was producing an array with a length of 3 (2 manifests and an "empty" manifest).  Hadn't seen it before, but decided to add this line to protect from other pages having the same issue.